setup.py

```diff
@@ -36,7 +36,13 @@ def find_version(*parts):
       package_data={
           'extra_geom.tests': ['dssc_geo_june19.h5', 'lpd_mar_18.h5'],
       },
+      entry_points={
+          'console_scripts': [
+              'extra-geom-from-karabo = extra_data.geom_from_karabo:main',
+          ]
+      },
       install_requires=[
+          'Karabo-proxy',
```

Comment:
Can we make this an optional dependency, using the `extras_require`  section below? If people are installing our tools in their own environment, I try to keep the number of dependencies minimal.

Reply:
Hmm, then I we also need to catch exception on import...

Reply:
If you want to, but I think it's also OK to just let the `ImportError` show in that case.